packages/safe-service-client/src/types/safeTransactionServiceTypes.ts

```diff
@@ -134,7 +134,7 @@ export type SafeMultisigTransactionResponse = {
   readonly dataDecoded?: string
   readonly confirmationsRequired: number
   readonly confirmations?: SafeMultisigConfirmationResponse[]
-  readonly signatures: string
+  readonly signatures?: string
```

Comment:
The endpoint can return `null` which is different than `undefined` 

![image](https://user-images.githubusercontent.com/2352112/135266975-670a39f9-b54b-4488-a9cf-b15532ff1ee8.png)

If you want to reflect the returned data type should be `signatures: string | null`

```suggestion
  readonly signatures: string | null
```

Reply:
@Uxio0 I believe the issue is also in swagger docs, right? 

https://safe-transaction.rinkeby.gnosis.io/

![image](https://user-images.githubusercontent.com/2352112/135267504-e8e10eab-d4a8-4266-a27a-2ec1e525009e.png)

Reply:
Should it be nullable? https://swagger.io/docs/specification/data-models/data-types/#null ?

Reply:
Probably it would be good to revisit the swagger docs for the other fields, cause at least `transactionHash` is also null while the transaction is not mined

Reply:
Thanks!

I talked with Uxío and he will make the field not required.

I will also generate all the types with [openapi-typescript](https://www.npmjs.com/package/openapi-typescript) so I will wait until this is updated: https://safe-transaction.rinkeby.gnosis.io/?format=openapi

Reply:
Yes, there's a typo on the docs, thanks for reporting